net/cryfs/Portfile

```diff
@@ -23,17 +24,24 @@ long_description          CryFS encrypts your files, so you can safely store the
 
 homepage                  https://www.cryfs.org/
 
-checksums                 rmd160  ba20af5cc2d77954b3a6f248b230860242cb690f \
-                          sha256  20348e6e2ff670505a14d4f8b2a9aa6104ce301d31ef50100cd03d17c1d7322b \
-                          size    10184695
+checksums                 rmd160  798106ac28621532695bf29ec39c47e64f5bf674 \
+                          sha256  a89ab8fea2d494b496867107ec0a3772fe606ebd71ef12152fcd233f463a2c00 \
+                          size    10419264
 
 extract.mkdir             yes
 
-depends_build-append      path:lib/libssl.dylib:openssl
+patchfiles                gitversion_python3_fix.diff
+post-patch {
+    reinplace "s|@@PYTHONBIN@@|${prefix}/bin/python3.10|g" ${worksrcpath}/src/gitversion/gitversion.cmake
+}
+
+depends_build-append      path:lib/libssl.dylib:openssl \
+                          port:range-v3 \
+                          port:spdlog \
+                          port:python310
 
 depends_lib-append        port:curl \
-                          port:libomp \
-                          port:osxfuse
+                          port:libomp
```

Comment:
is this actually working with the default Apple clang compilers? Shouldn't this use something like `compiler.openmp_version ???` and such - see [here](https://trac.macports.org/wiki/CompilerSelection)?

Reply:
yes, cryfs detects OpenMP installed with macports (or homebrew): https://github.com/cryfs/cryfs/commit/08ed96680876c960d5aab0aabebb25206f0bda06#diff-1902f39b31e47c36e9d15ad9d1008e5b95fb0088f11f535bb8b1444a53c5d815
On my box it uses `/usr/bin/clang++` (Apple clang version 13.1.6 (clang-1316.0.21.2)) with `-Xpreprocessor -fopenmp -I/opt/local/include/libomp/ /opt/local/lib/libomp/libomp.dylib`
therefore the `compiler.openmp_version` option is not needed, unless I'm missing something ?

Reply:
I am marking this request as resolved. feel free to re-open it if I misunderstood.